framework/core/js/src/common/Translator.tsx

```diff
@@ -88,4 +96,21 @@ export default class Translator {
 
     return id;
   }
+
+  /**
+   * Formats the time.
+   *
+   * The format of the time will be chosen by the following order:
+   * - The format ID defined in current locale.
+   * - The provided fallback format.
+   * - DayJS default format.
+   */
+  formatDateTime(time: Dayjs, id?: string, fallback?: string): string {
+    const formatCallback = id && this.dateTimeFormats.has(id) && this.dateTimeFormats.get(id);
+    if (formatCallback) {
+      const result = formatCallback(fallback);
+      if (result) return result;
+    }
+    return time.format(id && (this.translations[id] ?? fallback));
+  }
```

Comment:
My bad, let's remove the fallback parameter, it turns out it's not necessary 👍🏼 

```suggestion

  /**
   * Formats the time.
   *
   * The format of the time will be chosen by the following order:
   * - Custom format defined in the item list.
   * - The format defined in current locale.
   * - DayJS default format.
   */
  formatDateTime(time: Dayjs, id?: string): string {
    const formatCallback = id && this.dateTimeFormats.has(id) && this.dateTimeFormats.get(id);
    if (formatCallback) {
      const result = formatCallback(fallback);
      if (result) return result;
    }
    return time.format(id && (this.translations[id] ?? id));
  }
```